test/mocha.opts

```diff
@@ -2,5 +2,3 @@
 --reporter dot
 --recursive
 test/utils/setup.js
-test/integration/{,**/}*.spec.js
```

Comment:
Did you try the `only` thing? Now we have that config at three different locations, not great!